web/src/main/java/com/futurewei/alcor/web/entity/NetworkSegmentRangeWebRequestJson.java

```diff
@@ -0,0 +1,25 @@
+package com.futurewei.alcor.web.entity;
+
+import lombok.Data;
+
+import java.util.UUID;
+
+@Data
+public class NetworkSegmentRangeWebRequestJson {
```

Comment:
Thanks for adding those new entity classes in AlcorWeb.

Reply:
should it be a child class of CustomerResource?

Reply:
I make NetworkSegmentRangeWebRequest be a child class of CustomerResource ~
NetworkSegmentRangeWebRequestJson maybe not

Reply:
I see it is okay.